Utils.ps1

```diff
@@ -265,6 +576,13 @@ function Import-ConfigFile {
         Write-Error "IpConfig either missing or incorrect for lab $($lab.DevTestLabName).  Must be 'Public', 'Private', or 'Shared'"
     }
 
+    # Convert BastionEnabled to a boolean. If BastionEnabled property is not set, defaults to $false 
+    if ($lab.BastionEnabled) {
```

Comment:
Handle using and old config csv with no param `BastionEnabled`. Defaults to false.